Tests/API/Controllers/TraderManagementControllerTests.cs

```diff
@@ -0,0 +1,71 @@
+
+using Microsoft.AspNetCore.Mvc;
+using Application.Interfaces;
+using NUnit.Framework;
+using API.Controllers;
+using Domain.Entities;
+using Moq;
+
+
+namespace Tests.API.Controllers
+{
+    [TestFixture]
+    public class TraderManagementControllerTests
+    {
+        private Mock<ITraderManagement> _mockTraderManagement = null!;
+        private TraderManagementController _controller = null!;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockTraderManagement = new Mock<ITraderManagement>();
+            _controller = new TraderManagementController(_mockTraderManagement.Object);
+        }
```

Comment:
you can init it inside ctor, and use _mockTraderManagement.Reset() for reset the mock.